lib/pages/main/wallet_contents/settings/wallet_connect.dart

```diff
@@ -0,0 +1,264 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:qubic_wallet/config.dart';
+import 'package:qubic_wallet/di.dart';
+import 'package:qubic_wallet/flutter_flow/theme_paddings.dart';
+import 'package:qubic_wallet/helpers/global_snack_bar.dart';
+import 'package:qubic_wallet/l10n/l10n.dart';
+import 'package:qubic_wallet/models/wallet_connect.dart';
+import 'package:qubic_wallet/resources/secure_storage.dart';
+import 'package:qubic_wallet/services/qubic_hub_service.dart';
+import 'package:qubic_wallet/services/wallet_connect_service.dart';
+import 'package:qubic_wallet/stores/application_store.dart';
+import 'package:qubic_wallet/stores/qubic_hub_store.dart';
+import 'package:qubic_wallet/stores/settings_store.dart';
+import 'package:qubic_wallet/styles/edge_insets.dart';
+import 'package:qubic_wallet/styles/text_styles.dart';
+import 'package:qubic_wallet/styles/themed_controls.dart';
+import 'package:settings_ui/settings_ui.dart';
+import 'package:walletconnect_flutter_v2/walletconnect_flutter_v2.dart';
+
+class WalletConnect extends StatefulWidget {
+  const WalletConnect({super.key});
+
+  @override
+  // ignore: library_private_types_in_public_api
+  _AboutWalletState createState() => _AboutWalletState();
+}
+
+class _AboutWalletState extends State<WalletConnect> {
+  final ApplicationStore appStore = getIt<ApplicationStore>();
```

Comment:
why the class is named _AboutWalletState?

Reply:
Fixed, wrong cloning

Reply:
@Qubic-Hub  I see your answer but I don't see any commit pushed with the fix